bin/seed.js

```diff
@@ -7,6 +7,394 @@ const init = async () => {
     try {
         await db.sync({ force: true })
         await Category.create({ name: 'salty' })
+        const Puff = await Product.create({
```

Comment:
On line 3, you need to import the Product model

Reply:
Done!Thank you